onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2339,6 +2339,44 @@ void RegisterContribSchemas() {
         updateOutputShape(ctx, 0, output_shape);
       });
 
+  static const char* DisentangledAttention_TRT_ver1_doc =
+      R"DOC(Disentangled Attention TensorRT Plugin.)DOC";
+
+  ONNX_CONTRIB_OPERATOR_SCHEMA(DisentangledAttentionPlugin)
```

Comment:
Can this be named as DisentangledAttention_TRT (same style like EfficientNMS_TRT etc)?

Reply:
Changed as suggested